refinery/ui/source/js/user-file-browser/services/user-file.js

```diff
@@ -4,15 +4,24 @@
     .module('refineryApp')
     .factory('userFileService', userFileService);
 
-  userFileService.$inject = ['$resource', 'settings'];
+  // TODO: userFileBrowserFiltersCtrl.filters is what I need, but maybe I need a new service?
```

Comment:
@jkmarx : This is the point I'd like to fix when I come back to this tomorrow: userFileBrowserFiltersCtrl.filters has the information I need, but my guess is that I need to pull that out and put it in a service so I can access it here? (A controllers shouldn't be a dependency here?)

Let me know if I'm not thinking about this correctly?